### PR TITLE
[IOTDB-3756]Support mount template and query mount info

### PR DESCRIPTION
**Pull request by yunchan86**
Design Doc https://apache-iotdb.feishu.cn/docx/doxcnXewADHKWTpgJ5RnP4iyetd

Support mount template and query mount info
```sql
IoTDB> create schema template t1 (temperature FLOAT encoding=RLE, status BOOLEAN encoding=PLAIN compression=SNAPPY)
Msg: The statement is executed successfully.
IoTDB> create schema template t2 aligned (lat FLOAT encoding=Gorilla, lon FLOAT encoding=Gorilla);
Msg: The statement is executed successfully.
IoTDB> show schema templates;
+-------------+
|template name|
+-------------+
|           t1|
|           t2|
+-------------+
Total line number = 2
It costs 0.080s
IoTDB> show paths set schema template t1
+-----+
|paths|
+-----+
+-----+
Empty set.
It costs 0.044s
IoTDB> set schema template t1 to root.sg1.d1;
Msg: The statement is executed successfully.
IoTDB> show paths set schema template t1
+-----------+
|      paths|
+-----------+
|root.sg1.d1|
+-----------+
Total line number = 1
It costs 0.006s
IoTDB> set schema template t2 to root.sg1.d2;
Msg: The statement is executed successfully.
IoTDB> show paths set schema template t2
+-----------+
|      paths|
+-----------+
|root.sg1.d2|
+-----------+
Total line number = 1
It costs 0.004s
```

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ClusterSchemaManager.java [hunk 1]
```diff
@@ -373,6 +376,34 @@ public TGetTemplateResp getTemplate(String req) {
...
+   * @return
+   */
+  public List<String> getPathsSetTemplate(String templateName) {
+    GetPathsSetTemplatePlan getPathsSetTemplatePlan = new GetPathsSetTemplatePlan(templateName);
+    PathInfoResp resp =
+        (PathInfoResp) getConsensusManager().read(getPathsSetTemplatePlan).getDataset();
+    if (resp.getStatus().getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+      return resp.getPathList();
+    }
+    return null;
```
Comment (MarcosZyk) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ClusterSchemaManager.java:404]: If not success, shall we throw an exception or return a failure TSStatus?

**Review by MarcosZyk — COMMENTED** [review]
Nice work! Only one problem left. PTAL.

**Review by liuminghui233 — APPROVED** [review]
LGTM~

**Review by bigreybear — APPROVED** [review]
LGTM~

**Pull request merged**